distribution/docs/src/main/resources/_contents/_endpoints/content-endpoint-contents.adoc

```diff
@@ -0,0 +1,530 @@
+==== Content Endpoint
+
+The Content REST endpoint provides a CDR REST Retrieve v2.0-compliant DDF endpoint that allows clients to perform CRUD operations on the Content Repository using REST, a simple architectural style that performs communication using HTTP.
+
+The Content REST CRUD endpoint provides the capability to read, create, update, and delete content in the Content Repository, as well as create, update, and delete metacards in the catalog provider, i.e., the Metadata Catalog (MDC). Furthermore, this endpoint allows the client to perform the create/update/delete operations on just the Content Repository, just the MDC, or both in one operation.
```

Comment:
In the current documentation we sometimes refer to this as the Content Store as well. Are we set on Content Repository now? My only concern is that it is an overloaded term in an osgi/maven context.